doc/wiki/Installation:-Option-List.md

```diff
@@ -0,0 +1,147 @@
+|:---:|:---:|:---:|---|---|---|
+| `--openmp`      | `true`, `false` | `true` | Enable OpenMP (see [[MPI and OpenMP]]) | Must set the compilation flag `OPENMPFLAG` in [[configuration file \| Installation:-Machine-Configuration-File#3-Compilation-flags]] | <a name="--openmp"></a> `OPENMP` |
+| `--mpi`         | `true`, `false` | `false` | `true`: Enable load balancing using a space-filling curve (see [[MPI and OpenMP]]); `false`: Run GAMER in a serial mode, but OpenMP is still supported | May need to set `MPI_PATH` in [[configuration file \| Installation:-Machine-Configuration-File#1-Library-paths]] | <a name="--mpi"></a> `LOAD_BALANCE=HILBERT`, <a name="SERIAL"></a> `SERIAL` |
+| `--overlap_mpi` | `true`, `false` | `false` | Overlap MPI communication with computation. | Not supported yet!!! Must enable `--mpi`. | <a name="--overlap_mpi"></a> `OVERLAP_MPI` |
+| `--gpu`         | `true`, `false` | `false` | Enable GPU acceleration | Must specify `GPU_COMPUTE_CAPABILITY` in configuration file as well; may need to set `CUDA_PATH` in [[configuration file \| Installation:-Machine-Configuration-File#1-Library-paths]] | <a name="--gpu"></a> `GPU` |
```

Comment:
Change to "Must specify GPU_COMPUTE_CAPABILITY and may need to set CUDA_PATH in configuration file"